modules/performing-a-comparison-between-kubernetes-manifests.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-network-policies.adoc
+:_content-type: CONCEPT
+[id="performing-a-comparison-between-kubernetes-manifests_{context}"]
+= Performing a comparison between Kubernetes manifests
```

Comment:
```suggestion
= Comparing Kubernetes manifests to determine network policy changes
```

Reply:
Suggested title so that it ties in to the subject matter of the chapter (network policies)